clients/drcachesim/tracer/tracer.cpp

```diff
@@ -682,6 +685,73 @@ insert_conditional_skip_target(void *drcontext, instrlist_t *ilist, instr_t *whe
 #endif
 }
 
+static void
+insert_mode_comparison(void *drcontext, instrlist_t *ilist, instr_t *where,
+                       reg_id_t reg_ptr, void *addr, uint slot)
+{
```

Comment:
(OK there's nothing new in this function's code: looks like it was moved without change.  I wish github could recognize that and confirm it in the visual diff view.)